test/system/application-scans/java.spec.ts

```diff
@@ -28,142 +28,368 @@ describe("jar binaries scanning", () => {
       digest: expect.any(String),
     };
 
-    describe("with all needed CLI flags (app-vulns and shaded-jars-depth)", () => {
-      beforeAll(async () => {
-        // Arrange
-        fixturePath = getFixture(
-          "docker-archives/docker-save/java-uberjar.tar",
-        );
-        const imageNameAndTag = `docker-archive:${fixturePath}`;
+    // TODO: deprecate --shaded-jars-depth and leave only --nested-jars-depth
+    describe("--shaded-jars-depth", () => {
```

Comment:
everything is nested behind this new describe. "Hide whitespace" for a better view.